packages/parser/src/actions/CombatAdvancedAction.ts

```diff
@@ -40,7 +40,7 @@ export class CombatAdvancedAction extends CombatAction {
   constructor(logLine: ILogLine, wowVersion: WowVersion) {
     super(logLine);
     if (!CombatAdvancedAction.supports(logLine)) {
-      throw new Error('event not supported');
+      throw new Error('x event not supported');
```

Comment:
what does x mean here? should we use a consistent format like in CombatAction.ts?